### PR TITLE
Helm install with name

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

**Comment by jonahjon** [comment]
It looks like we still keep the digest here right?

**Comment by jonahjon** [comment]
Looks good but the title is a bit confusing

File: api/v1alpha1/packagebundle_test.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ func TestPackageBundle_Find(t *testing.T) {
 	expected := api.PackageOCISource{
 		Registry:   "public.ecr.aws/l0g8r8j6",
 		Repository: "eks-anywhere-test",
+		Name:       "v0.1.0",
```
Comment (jonahjon) [inline api/v1alpha1/packagebundle_test.go:37]: remove the v in front of 0.1.0 here I think

Comment (TerryHowe) [reply api/v1alpha1/packagebundle_test.go:37]: The v in the version is a problem with the helm Chart.yaml and this is just test data anyway that isn't even used

**Comment by TerryHowe** [comment]
> Looks good but the title is a bit confusing

Fair enough, I changed the title to "Helm install with name"

File: api/v1alpha1/packagebundle.go [hunk 2]
```diff
@@ -8,6 +8,7 @@ import (
 type PackageOCISource struct {
 	Registry   string `json:"registry"`
 	Repository string `json:"repository"`
+	Name       string `json:"name"`
```
[the remark this answers is not in the document]
Comment (TerryHowe) [reply api/v1alpha1/packagebundle.go:11]: `Name` is from `Name` in the bundle

**Comment by ewollesen** [comment]
I don't have a problem merging this, but since we just found a way to sidestep helm and it's semver stuff, I'm wondering if we need this anymore?

**Comment by TerryHowe** [comment]
> I don't have a problem merging this, but since we just found a way to sidestep helm and it's semver stuff, I'm wondering if we need this anymore?


Agreed, hopefully we don't need this, but I'll leave it up in case the other approach runs into a snag.

**Pull request closed without merging**